### ISSUE TITLE
Please build images for other architectures

### ISSUE THREAD

**Issue by corbinlc** [ParrotSec/docker-images#4]
We have had a few requests to support Parrot in the UserLAnd Android app.  We have started making our lives easier by creating our base images from other's dockerhub images.  Looking at the parrotsec images, I see that they are only for x86_64.  Would you consider building for x86, arm32v7 and arm64v8 as well?

**Comment by Free-Radical** [comment]
I'm running Docker on OSX to build parrot but its crashing at command listed below (I'm a Docker newbie).  I am using build directions from https://www.parrotsec.org/docs/getting-started/install-docker/#docker for the metasploit version 

```
$ docker build -t metasploit[:version] metasploit

invalid argument "metasploit[:version]" for "-t, --tag" flag: invalid reference format
See 'docker build --help'
```
Advice?

**Comment by PalinuroSec** [comment]
yes, i can take it in consideration, but i have never created docker images for foreign architectures.

by the way if you are building a smartphone solution that ships parrot, we would be interested in helping you and officially support it

**Comment by vdbaan** [comment]
Would you consider using the same configuration in the security folder as you use in core. 
- lts-amd64
- lts-arm64
- lts-armhf
- lts-i386
- rolling-amd64
- rolling-i386

and perhaps adding the following to core and security:
- rolling-arm64

**Comment by tidux** [comment]
This issue has become much more interesting with the general availability of high performance 64-bit ARM hardware from several vendors, including Apple and AWS, and the Raspberry Pi 4(00).  Images for arm64 are table stakes now.

**Comment by PalinuroSec** [comment]
with parrot 5.0 finally out, the rolling branch is deprecated and the lts one is the only supported branch. all the docker containers are built with multiarch support, and we also offer plain rootfs tarballs available here https://deb.parrot.sh/parrot/iso/5.0/

the build process of the arm images is done on amd64 servers with qemu and it needs some testing on native arm64 environments

our documentation needs an update as well

**Comment by vdbaan** [comment]
There is still an issue with the docker arm64 image. As you can see docker states that the requested image is amd64.
(I am running on Apple M1)

```
$ docker run -it --rm  parrotsec/core:lts-arm64
Unable to find image 'parrotsec/core:lts-arm64' locally
lts-arm64: Pulling from parrotsec/core
f403af20c30d: Pull complete
7036e13e24f2: Pull complete
f47b87e12274: Pull complete
Digest: sha256:3c3b27fac2c263a0fb074b012cd7cec05870baed6285097c9751e213270174e4
Status: Downloaded newer image for parrotsec/core:lts-arm64
WARNING: The requested image's platform (linux/amd64) does not match the detected host platform (linux/arm64/v8) and no specific platform was requested
┌─[root@dd9b8836986a]─[/]
└──╼ # 
```

And when specifying the platform I get the following:
```
$ docker run -it --rm --platform linux/arm64 parrotsec/core:lts-arm64
Unable to find image 'parrotsec/core:lts-arm64' locally
lts-arm64: Pulling from parrotsec/core
Digest: sha256:3c3b27fac2c263a0fb074b012cd7cec05870baed6285097c9751e213270174e4
Status: Image is up to date for parrotsec/core:lts-arm64
docker: Error response from daemon: image with reference parrotsec/core:lts-arm64 was found but does not match the specified platform: wanted linux/arm64, actual: linux/amd64.
```
